src/services/task_services.py

```
@@ -0,0 +1,325 @@
+        Returns:
+        Task: The task with the specified ID.
+        """
+        try:
+            statement = select(Task).where(Task.id == task_id)
+            result = await self.session.execute(statement)
+            task = result.scalars().first()
+            return task
+        except SQLAlchemyError:
+            return None
+
+    async def filter_tasks(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)